src/components/Header.jsx

```
@@ -80,11 +80,11 @@ const Header = () => {
           <Nav flexDirection="flex-col" menuOpen={menuOpen} />
           <div className="flex flex-col space-y-2.5 mt-5 mx-auto max-w-2/3 ">
             <Button
-              children="Log In"
+              children="LOG IN"
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)